examples/beam/src/main/java/edu/snu/nemo/examples/beam/AlternatingLeastSquare.java

```diff
@@ -326,11 +328,22 @@ public static void main(final String[] args) {
     final Integer numFeatures = Integer.parseInt(args[1]);
     final Integer numItr = Integer.parseInt(args[2]);
     final Double lambda;
-    if (args.length > 4) {
+    if (args.length > 3) {
       lambda = Double.parseDouble(args[3]);
     } else {
       lambda = 0.05;
     }
+    final String outputFilePath;
+    boolean checkOutput = false;
```

Comment:
remove this argument, and simply check if the `outputFilePath` argument is given or not?

Reply:
Let me know if this makes sense.

Reply:
I've remove the argument named `checkOutput` at `AlternatingLeastSquareITCase`. `checkOutput` in here is just for private use.

Reply:
Got it. Thanks!